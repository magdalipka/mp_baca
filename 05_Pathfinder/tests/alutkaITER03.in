8 5
0 0 1 0 0 1 0 0
1 0 1 1 0 1 0 0
1 0 0 1 0 0 0 0
1 0 0 0 0 1 0 0
0 0 1 1 1 1 0 0
1
i 0 4 7 0

