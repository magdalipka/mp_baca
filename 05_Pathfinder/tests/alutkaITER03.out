i E S S E S E E N E E E S W S E
